pymongo/encryption.py

```diff
@@ -682,6 +704,74 @@ def encrypt(
                 key_alt_name=key_alt_name,
                 query_type=query_type,
                 contention_factor=contention_factor,
+                range_opts=range_options,
+            )
+            return decode(encrypted_doc)["v"]  # type: ignore[index]
+
+    def encrypt_expression(
```

Comment:
Ideally encrypt() and encrypt_expression() can be refactored to share the same logic since the only difference is the type of the return value and passing is_expression=True/False to pymongocrypt.